*** Begin Patch
*** Update File: c:\Users\volod\keebcult-articles\docs\Что купить в бюджете до 15 тысяч (2025).md
@@ -22,94 +22,151 @@

Характеристики:

-- Материал корпуса: ABS пластик
-- Количество клавиш: 68
-- Тип крепления: Tray mount
-- Плейт: Алюминиевый
-- Переключатели: Линейные HMX
-- Хотсвап: Отсуствует
-- Кейкапы: ABS
-- Цвета: Серый
-- Подключение: Bluetooth 5.0/2.4g/Провод
-- Подсветка: Отсутствует
-- Аккумулятор: AAA Батарейки
-- Софт: VIA
-- Вес: 770гр
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>ABS пластик</dd>
+  <dt>Количество клавиш</dt>
+  <dd>68</dd>
+  <dt>Тип крепления</dt>
+  <dd>Tray mount</dd>
+  <dt>Плейт</dt>
+  <dd>Алюминиевый</dd>
+  <dt>Переключатели</dt>
+  <dd>Линейные HMX</dd>
+  <dt>Хотсвап</dt>
+  <dd>Отсуствует</dd>
+  <dt>Кейкапы</dt>
+  <dd>ABS</dd>
+  <dt>Цвета</dt>
+  <dd>Серый</dd>
+  <dt>Подключение</dt>
+  <dd>Bluetooth 5.0/2.4g/Провод</dd>
+  <dt>Подсветка</dt>
+  <dd>Отсутствует</dd>
+  <dt>Аккумулятор</dt>
+  <dd>AAA Батарейки</dd>
+  <dt>Софт</dt>
+  <dd>VIA</dd>
+  <dt>Вес</dt>
+  <dd>770гр</dd>
+</dl>

Почему эта клавиатура в списке? Довольно интересный пребилд с неплохими свитчами и отличным внешним видом, а что еще нужно в этом бюджете? Необычный вариант для тех, кто хочет необычненького. Есть проводная версия подешевле, есть беспроводная версия на мизинчиковых батарейках подороже, выбор за вами.

-[Ссылка (али)](https://aliexpress.ru/item/1005008541865816.html?sku_id=12000045635552513&srcSns=sns_More&businessType=ProductDetail&spreadType=socialShare&tt=MG&utm_medium=sharing)
-
-[Тайпинг](https://www.youtube.com/watch?v=jkdt9Opd2_o&ab_channel=mumukeebs)
+[Ссылка (али)](https://aliexpress.ru/item/1005008541865816.html?sku_id=12000045635552513&srcSns=sns_More&businessType=ProductDetail&spreadType=socialShare&tt=MG&utm_medium=sharing){: .btn .btn-purple }
+
+[Тайпинг](https://www.youtube.com/watch?v=jkdt9Opd2_o&ab_channel=mumukeebs){: .btn .btn-blue }

### Cidoo qk61

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%202.png)

-- Материал корпуса: ABS Пластик
-- Количество клавиш: 61
-- Тип крепления: Gasket mount
-- Плейт: Поликарбонат
-- Переключатели: CIDOO Pearl White
-- Хотсвап: 3/5 Pin
-- Кейкапы: PBT Cherry
-- Цвета: Белая/Черная
-- Подключение: Bluetooth 5.0/2.4g/Провод
-- Подсветка: RGB
-- Аккумулятор: 3000 mAh
-- Софт: VIA
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>ABS Пластик</dd>
+  <dt>Количество клавиш</dt>
+  <dd>61</dd>
+  <dt>Тип крепления</dt>
+  <dd>Gasket mount</dd>
+  <dt>Плейт</dt>
+  <dd>Поликарбонат</dd>
+  <dt>Переключатели</dt>
+  <dd>CIDOO Pearl White</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 Pin</dd>
+  <dt>Кейкапы</dt>
+  <dd>PBT Cherry</dd>
+  <dt>Цвета</dt>
+  <dd>Белая/Черная</dd>
+  <dt>Подключение</dt>
+  <dd>Bluetooth 5.0/2.4g/Провод</dd>
+  <dt>Подсветка</dt>
+  <dd>RGB</dd>
+  <dt>Аккумулятор</dt>
+  <dd>3000 mAh</dd>
+  <dt>Софт</dt>
+  <dd>VIA</dd>
+</dl>

Популярная клавиатура, о которой не нужно много говорить, просто приятная пластиковая шисятка с поддержкой VIA. 

-[Ссылка (озон)](https://www.ozon.ru/product/igrovaya-klaviatura-cidoo-qk-61-v2-russkaya-raskladka-pearl-white-linear-switches-belaya-1858683236/?at=A6tG5PGJ5c00ZmEmHxBZ3lYuqkQA3piDpLl8LFMG1zQy&keywords=cidoo+qk61)
-
-[Тайпинг](https://www.youtube.com/shorts/ZuN_WU-hcq8)
+[Ссылка (озон)](https://www.ozon.ru/product/igrovaya-klaviatura-cidoo-qk-61-v2-russkaya-raskladka-pearl-white-linear-switches-belaya-1858683236/?at=A6tG5PGJ5c00ZmEmHxBZ3lYuqkQA3piDpLl8LFMG1zQy&keywords=cidoo+qk61){: .btn .btn-purple }
+
+[Тайпинг](https://www.youtube.com/shorts/ZuN_WU-hcq8){: .btn .btn-blue }

### Feker ik65

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%203.png)

-- Материал корпуса: ABS Пластик
-- Количество клавиш: 61
-- Тип крепления: Gasket mount
-- Плейт: Поликарбонат
-- Переключатели: Feker white jade/matcha
-- Хотсвап: 3/5 Pin
-- Кейкапы: PBT Cherry
-- Цвета: Белая/Коричневая
-- Подключение: Bluetooth 5.0/2.4g/Провод
-- Подсветка: RGB
-- Аккумулятор: 4000 mAh
-- Софт: VIA
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>ABS Пластик</dd>
+  <dt>Количество клавиш</dt>
+  <dd>61</dd>
+  <dt>Тип крепления</dt>
+  <dd>Gasket mount</dd>
+  <dt>Плейт</dt>
+  <dd>Поликарбонат</dd>
+  <dt>Переключатели</dt>
+  <dd>Feker white jade/matcha</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 Pin</dd>
+  <dt>Кейкапы</dt>
+  <dd>PBT Cherry</dd>
+  <dt>Цвета</dt>
+  <dd>Белая/Коричневая</dd>
+  <dt>Подключение</dt>
+  <dd>Bluetooth 5.0/2.4g/Провод</dd>
+  <dt>Подсветка</dt>
+  <dd>RGB</dd>
+  <dt>Аккумулятор</dt>
+  <dd>4000 mAh</dd>
+  <dt>Софт</dt>
+  <dd>VIA</dd>
+</dl>

Гмк67 помним? Помним. А это, по сути, собранная гмк, но с Via, что дает отличное преимущество на фоне гмк. Свитчи смазаны, стабы далеко не самые паршивые, нормальные кепки. Хороший вариант, если хочется шисятпяточку.

-[Ссылка (али)](https://aliexpress.ru/item/1005005481066386.html?sku_id=12000037043795937&spm=a2g2w.productlist.search_results.0.53262c4fjG4vx5)
-
-[Тайпинг](https://www.youtube.com/shorts/MbIdUQj4B_E)
+[Ссылка (али)](https://aliexpress.ru/item/1005005481066386.html?sku_id=12000037043795937&spm=a2g2w.productlist.search_results.0.53262c4fjG4vx5){: .btn .btn-purple }
+
+[Тайпинг](https://www.youtube.com/shorts/MbIdUQj4B_E){: .btn .btn-blue }

### Aula f65

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%204.png)

-- Материал корпуса: ABS Пластик
-- Количество клавиш: 67
-- Тип крепления: Gasket mount
-- Плейт: Поликарбонат
-- Переключатели: На выбор в зависимости от цвета/продавца
-- Хотсвап: 3/5 Pin
-- Кейкапы: PBT Cherry
-- Цвета: Белая/Коричневая
-- Подключение: Bluetooth 5.0/2.4g/Провод
-- Подсветка: RGB
-- Аккумулятор: 4000 mAh
-- Софт: VIA (?)
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>ABS Пластик</dd>
+  <dt>Количество клавиш</dt>
+  <dd>67</dd>
+  <dt>Тип крепления</dt>
+  <dd>Gasket mount</dd>
+  <dt>Плейт</dt>
+  <dd>Поликарбонат</dd>
+  <dt>Переключатели</dt>
+  <dd>На выбор в зависимости от цвета/продавца</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 Pin</dd>
+  <dt>Кейкапы</dt>
+  <dd>PBT Cherry</dd>
+  <dt>Цвета</dt>
+  <dd>Белая/Коричневая</dd>
+  <dt>Подключение</dt>
+  <dd>Bluetooth 5.0/2.4g/Провод</dd>
+  <dt>Подсветка</dt>
+  <dd>RGB</dd>
+  <dt>Аккумулятор</dt>
+  <dd>4000 mAh</dd>
+  <dt>Софт</dt>
+  <dd>VIA (?)</dd>
+</dl>

Просто альтернативный вариант 65% клавиатуры для потребителя, которому обязательно нужна кириллица. Заявляется некоторыми продавцами поддержка Via, но так как ее нет в F75, то и тут бы я сильно на это не рассчитывал, софт скорее всего проприетарный. А так - за 3000 вечно деревянных не самый плохой вариант.

-[Ссылка (озон)](https://www.ozon.ru/product/aula-klaviatura-klaviatura-mehanicheskaya-f65-priemnik-2-4g-bluetooth-type-c-4000mah-russkaya-1896366647/?at=WPtN8YoD7Ur9jWr3TnBEy7WiK2oWlLiqzXWlfR7JWgN&keywords=aula+f65)
-
-[Тайпинг](https://www.youtube.com/watch?v=hvwmpmXHulY&ab_channel=Tech4Gamers)
+[Ссылка (озон)](https://www.ozon.ru/product/aula-klaviatura-klaviatura-mehanicheskaya-f65-priemnik-2-4g-bluetooth-type-c-4000mah-russkaya-1896366647/?at=WPtN8YoD7Ur9jWr3TnBEy7WiK2oWlLiqzXWlfR7JWgN&keywords=aula+f65){: .btn .btn-purple }
+
+[Тайпинг](https://www.youtube.com/watch?v=hvwmpmXHulY&ab_channel=Tech4Gamers){: .btn .btn-blue }

### Royal Kludge r65

@@ -117,24 +174,38 @@

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%205.png)

-- Материал корпуса: ABS Пластик
-- Количество клавиш: 66
-- Тип крепления: Gasket mount
-- Плейт: Поликарбонат
-- Переключатели: RK Chartreuse/brown
-- Хотсвап: 3/5 Pin
-- Кейкапы: PBT Cherry
-- Цвета: Зеленая/Коричневая
-- Подключение: Bluetooth 5.0/2.4g/Провод
-- Подсветка: RGB
-- Аккумулятор: 4000 mAh
-- Софт: VIA
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>ABS Пластик</dd>
+  <dt>Количество клавиш</dt>
+  <dd>66</dd>
+  <dt>Тип крепления</dt>
+  <dd>Gasket mount</dd>
+  <dt>Плейт</dt>
+  <dd>Поликарбонат</dd>
+  <dt>Переключатели</dt>
+  <dd>RK Chartreuse/brown</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 Pin</dd>
+  <dt>Кейкапы</dt>
+  <dd>PBT Cherry</dd>
+  <dt>Цвета</dt>
+  <dd>Зеленая/Коричневая</dd>
+  <dt>Подключение</dt>
+  <dd>Bluetooth 5.0/2.4g/Провод</dd>
+  <dt>Подсветка</dt>
+  <dd>RGB</dd>
+  <dt>Аккумулятор</dt>
+  <dd>4000 mAh</dd>
+  <dt>Софт</dt>
+  <dd>VIA</dd>
+</dl>

По сути, тот же фекер, та же смысловая нагрузка, то же наличие виа. На самом деле нет особого смысла описывать отдельно рк, потому что буквально ряд rk - feker - aula это все довольно похожие между собой клавиатуры. Есть возможность взять здесь и сейчас в днс без ожидания, иногда кому то это важно.

[Ссылка (озон)](https://www.ozon.ru/product/royal-kludge-igrovaya-klaviatura-provodnaya-r65-provodnaya-66-klavish-gasket-mount-phantom-rk-brown-1846762467/?at=16tLXmWqyuw60gpRHG3N80Xs176JA6fxXL1qmizEMO6O&keywords=royal+kludge+r75) / [Ссылка (днс)](https://www.dns-shop.ru/product/7feaf1e1b13fbffc/klaviatura-provodnaa-royal-kludge-r65/?utm_medium=organic&utm_source=google&utm_referrer=https%3A%2F%2Fwww.google.com%2F)

-[Тайпинг](https://www.youtube.com/shorts/2YwRFf7aKd8)
+[Тайпинг](https://www.youtube.com/shorts/2YwRFf7aKd8){: .btn .btn-purple }

## **75%:**

@@ -146,67 +217,107 @@

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%208.png)

-- Материал корпуса: Алюминий
-- Количество клавиш: 81
-- Тип крепления: Gasket mount
-- Плейт: Полипропилен
-- Переключатели: Leobog Greywood V3/Nimbus V3/Block v2
-- Хотсвап: 3/5 Pin
-- Кейкапы: PBT Cherry/PBT SOA
-- Цвета: Белый/розовый/сиреневый
-- Подключение: Bluetooth 5.0/2.4g/Провод
-- Подсветка: RGB
-- Аккумулятор: 4000 mAh
-- Софт: Проприетарный
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>Алюминий</dd>
+  <dt>Количество клавиш</dt>
+  <dd>81</dd>
+  <dt>Тип крепления</dt>
+  <dd>Gasket mount</dd>
+  <dt>Плейт</dt>
+  <dd>Полипропилен</dd>
+  <dt>Переключатели</dt>
+  <dd>Leobog Greywood V3/Nimbus V3/Block v2</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 Pin</dd>
+  <dt>Кейкапы</dt>
+  <dd>PBT Cherry/PBT SOA</dd>
+  <dt>Цвета</dt>
+  <dd>Белый/розовый/сиреневый</dd>
+  <dt>Подключение</dt>
+  <dd>Bluetooth 5.0/2.4g/Провод</dd>
+  <dt>Подсветка</dt>
+  <dd>RGB</dd>
+  <dt>Аккумулятор</dt>
+  <dd>4000 mAh</dd>
+  <dt>Софт</dt>
+  <dd>Проприетарный</dd>
+</dl>

Подешевел, потому стал отличным приобретением. Неплохого качества кейкапы, приятные свитчи, хорошо смазанные в стоке, классные стабилизаторы и, в целом, это первая алюминиевая клавиатура в данном гайде. Приятный бонус - полированный бейджик с космонавтом. На мой вкус, вставку на knob можно бы и заменить, благо, они есть в продаже, иначе родная выглядит чересчур вырвиглазно. 

-[Ссылка (озон)](https://www.ozon.ru/product/leobog-igrovaya-klaviatura-provodnaya-hi75-milky-white-purple-grey-wood-v3-switches-cherry-1680264186/?at=ywtA3WVLvsyW6qW0sp7pZG7t0vWqg5IkE6vrPhno860M&keywords=Leobog+hi75)
-
-[Тайпинг](https://www.youtube.com/shorts/bUhN7S0dMfE)
+[Ссылка (озон)](https://www.ozon.ru/product/leobog-igrovaya-klaviatura-provodnaya-hi75-milky-white-purple-grey-wood-v3-switches-cherry-1680264186/?at=ywtA3WVLvsyW6qW0sp7pZG7t0vWqg5IkE6vrPhno860M&keywords=Leobog+hi75){: .btn .btn-blue }
+
+[Тайпинг](https://www.youtube.com/shorts/bUhN7S0dMfE){: .btn .btn-purple }

### Aula f75/Ajazz ak820 pro/Royal Kludge r75

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%209.png)

-- Материал корпуса: ABS пластик
-- Количество клавиш: 78\81\81
-- Тип крепления: gasket mount
-- Хотсвап: присутствует
-- Плейт: поликарбонат
-- Подключение: Tri-mode\Tri-mode\Wired
-- Подсветка: Есть
-- Совместимость: MAC/Windows/IOS/Android
-- Софт: Проприетарный
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>ABS пластик</dd>
+  <dt>Количество клавиш</dt>
+  <dd>78\81\81</dd>
+  <dt>Тип крепления</dt>
+  <dd>gasket mount</dd>
+  <dt>Хотсвап</dt>
+  <dd>присутствует</dd>
+  <dt>Плейт</dt>
+  <dd>поликарбонат</dd>
+  <dt>Подключение</dt>
+  <dd>Tri-mode\Tri-mode\Wired</dd>
+  <dt>Подсветка</dt>
+  <dd>Есть</dd>
+  <dt>Совместимость</dt>
+  <dd>MAC/Windows/IOS/Android</dd>
+  <dt>Софт</dt>
+  <dd>Проприетарный</dd>
+</dl>

Три популярных брата - акробата, похожих между собой, а потому лишь скажу, что на фоне всех трех самой популярной является аула, самой, на мой взгляд, интересной является аджазз (он же единственный с экранчиком, удивительно, но иногда кому то это бывает принципиально), но и он же и единственный без кириллицы. Тут можно брать то, к чему лежит душа, по сути, одно и тоже.

-[Ссылка f75 (озон)](https://www.ozon.ru/product/klaviatura-besprovodnaya-mehanicheskaya-f75-2-4g-priemnik-bluetooth-type-c-4000mah-16-1875474325/?at=BrtzNm28QfQ9BvW0C65GmgMCJ8OjBBu957QO6TLBYRmo&keywords=aula+f65) | [тайпинг](https://www.youtube.com/shorts/Sl0dBeNu4Ik)
-
-[Ссылка r75 (озон)](https://www.ozon.ru/product/royal-kludge-klaviatura-provodnaya-r75-rk-silver-russkaya-raskladka-chernyy-1704546250/?at=99trjK72yTwMB53GurmOqQBcBoA8jrtARA3MMsP1NKjZ&keywords=Royal+Kludge+r75) | [тайпинг](https://www.youtube.com/shorts/FibZyZXbSlI)
-
-[Ссылка ak820 (озон)](https://www.ozon.ru/product/ajazz-igrovaya-klaviatura-besprovodnaya-ak820-goryachaya-zamena-russkaya-versiya-raskladki-1545499727/?at=08tYzWPmOc1RVG3rhRPwKojTy9nBM3upDzq8QtvQRRxL&keywords=Ajazz+ak820%2Fpro) | [тайпинг](https://www.youtube.com/watch?v=kqEBf2bNj7s&ab_channel=Jur1cek)
+[Ссылка f75 (озон)](https://www.ozon.ru/product/klaviatura-besprovodnaya-mehanicheskaya-f75-2-4g-priemnik-bluetooth-type-c-4000mah-16-1875474325/?at=BrtzNm28QfQ9BvW0C65GmgMCJ8OjBBu957QO6TLBYRmo&keywords=aula+f65){: .btn .btn-blue }
+[тайпинг](https://www.youtube.com/shorts/Sl0dBeNu4Ik){: .btn .btn-purple }
+
+[Ссылка r75 (озон)](https://www.ozon.ru/product/royal-kludge-klaviatura-provodnaya-r75-rk-silver-russkaya-raskladka-chernyy-1704546250/?at=99trjK72yTwMB53GurmOqQBcBoA8jrtARA3MMsP1NKjZ&keywords=Royal+Kludge+r75){: .btn .btn-blue }
+[тайпинг](https://www.youtube.com/shorts/FibZyZXbSlI){: .btn .btn-purple }
+
+[Ссылка ak820 (озон)](https://www.ozon.ru/product/ajazz-igrovaya-klaviatura-besprovodnaya-ak820-goryachaya-zamena-russkaya-versiya-raskladki-1545499727/?at=08tYzWPmOc1RVG3rhRPwKojTy9nBM3upDzq8QtvQRRxL&keywords=Ajazz+ak820%2Fpro){: .btn .btn-blue }
+[тайпинг](https://www.youtube.com/watch?v=kqEBf2bNj7s&ab_channel=Jur1cek){: .btn .btn-purple }

### Mchose g75

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2010.png)

-- Материал корпуса: ABS пластик
-- Количество клавиш: 75
-- Тип крепления: gasket mount
-- Свитчи: Несколько вариантов на выбор от КТТ
-- Хотсвап: присутствует
-- Плейт: поликарбонат
-- Подключение: Tri-mode\Tri-mode\Wired
-- Подсветка: Есть
-- Совместимость: MAC/Windows/IOS/Android
-- Софт: Проприетарный
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>ABS пластик</dd>
+  <dt>Количество клавиш</dt>
+  <dd>75</dd>
+  <dt>Тип крепления</dt>
+  <dd>gasket mount</dd>
+  <dt>Свитчи</dt>
+  <dd>Несколько вариантов на выбор от КТТ</dd>
+  <dt>Хотсвап</dt>
+  <dd>присутствует</dd>
+  <dt>Плейт</dt>
+  <dd>поликарбонат</dd>
+  <dt>Подключение</dt>
+  <dd>Tri-mode\Tri-mode\Wired</dd>
+  <dt>Подсветка</dt>
+  <dd>Есть</dd>
+  <dt>Совместимость</dt>
+  <dd>MAC/Windows/IOS/Android</dd>
+  <dt>Софт</dt>
+  <dd>Проприетарный</dd>
+</dl>

Новомодная калотурочка, которая смотрится довольно интересно. 8000 махов аккумулятор, интересные свитчи, да и приятная цена. Что еще нужно от клавиатуры? Хороший вариант, возможно, чуть более интересный на фоне вариантов выше.

-[Ссылка (озон)](https://www.ozon.ru/product/mchose-igrovaya-klaviatura-besprovodnaya-g75-pro-angliyskaya-raskladka-siniy-1788868953/?at=57tw1KN6mcD0nzkOhy7WJ4WUk3zwG2C19mB9PIvD8y5K&keywords=Mchose+g75)
-
-[Тайпинг](https://www.youtube.com/shorts/wzZtDpl6Vic)
+[Ссылка (озон)](https://www.ozon.ru/product/mchose-igrovaya-klaviatura-besprovodnaya-g75-pro-angliyskaya-raskladka-siniy-1788868953/?at=57tw1KN6mcD0nzkOhy7WJ4WUk3zwG2C19mB9PIvD8y5K&keywords=Mchose+g75){: .btn .btn-blue }
+
+[Тайпинг](https://www.youtube.com/shorts/wzZtDpl6Vic){: .btn .btn-purple }

## **80% (TKL):**

@@ -216,21 +327,32 @@

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2012.png)

-- Материал корпуса: ABS пластик
-- Количество клавиш: 87
-- Тип крепления: Gasket mount
-- Свитчи: rk cream/rk brown
-- Хотсвап: 3/5 pin
-- Плейт: Поликарбонат
-- Подключение: bt/2.4g/провод
-- Подсветка: RGB
-- Софт: Via
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>ABS пластик</dd>
+  <dt>Количество клавиш</dt>
+  <dd>87</dd>
+  <dt>Тип крепления</dt>
+  <dd>Gasket mount</dd>
+  <dt>Свитчи</dt>
+  <dd>rk cream/rk brown</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Плейт</dt>
+  <dd>Поликарбонат</dd>
+  <dt>Подключение</dt>
+  <dd>bt/2.4g/провод</dd>
+  <dt>Подсветка</dt>
+  <dd>RGB</dd>
+  <dt>Софт</dt>
+  <dd>Via</dd>
+</dl>

Интересная тклка, где можно поставить knob, либо заменить его свитчем. Интересное решение, но клавиатура, по своей сути, довольно обычная. Из плюсов - хорошие свитчи и качество сборки.

-[Ссылка (озон)](https://www.ozon.ru/product/klassicheskaya-tkl-80-mehanicheskaya-klaviatura-royal-kludge-rk-r87-pro-provodnaya-87-klavish-gasket-1816600944/?at=lRt6lK4XJT9Rp0LI782n4JcYwYqrquV0YgJDUE8gADn&keywords=royal+kludge+rk87pro)
-
-[Тайпинг](https://www.youtube.com/watch?v=H-LwdZ1dZLI&ab_channel=VoldemarHardware)
+[Ссылка (озон)](https://www.ozon.ru/product/klassicheskaya-tkl-80-mehanicheskaya-klaviatura-royal-kludge-rk-r87-pro-provodnaya-87-klavish-gasket-1816600944/?at=lRt6lK4XJT9Rp0LI782n4JcYwYqrquV0YgJDUE8gADn&keywords=royal+kludge+rk87pro){: .btn .btn-blue }
+
+[Тайпинг](https://www.youtube.com/watch?v=H-LwdZ1dZLI&ab_channel=VoldemarHardware){: .btn .btn-purple }

### Ajazz ak870 pro

@@ -238,41 +360,63 @@

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2014.png)

-- Материал корпуса: ABS Пластик
-- Количество клавиш: 87
-- Тип крепления: gasket mount
-- Свитчи: На выбор
-- Хотсвап: 3/5 pin
-- Плейт: Поликарбонат
-- Подключение: bt/2.4g/провод
-- Подсветка: rgb
-- Софт: Проприетарный
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>ABS Пластик</dd>
+  <dt>Количество клавиш</dt>
+  <dd>87</dd>
+  <dt>Тип крепления</dt>
+  <dd>gasket mount</dd>
+  <dt>Свитчи</dt>
+  <dd>На выбор</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Плейт</dt>
+  <dd>Поликарбонат</dd>
+  <dt>Подключение</dt>
+  <dd>bt/2.4g/провод</dd>
+  <dt>Подсветка</dt>
+  <dd>rgb</dd>
+  <dt>Софт</dt>
+  <dd>Проприетарный</dd>
+</dl>

Тут подобная рк история, только окромя кноба снимается/ставится еще и экран. Хорошие свитчи, хорошие кепки, отличный вариант тклки за свои деньги, больше сказать нечего.

-[Ссылка (озон)](https://www.ozon.ru/product/ajazz-igrovaya-klaviatura-besprovodnaya-ak870-goryachaya-zamena-angliyskaya-raskladka-siniy-1552037304/?at=DqtDZWx6XUBvkjnkhq89N6oS8mgw01smEwBL0umvQp36&keywords=Ajazz+ak870)
-
-[Тайпинг](https://www.youtube.com/shorts/k3Y6AowoDo8)
+[Ссылка (озон)](https://www.ozon.ru/product/ajazz-igrovaya-klaviatura-besprovodnaya-ak870-goryachaya-zamena-angliyskaya-raskladka-siniy-1552037304/?at=DqtDZWx6XUBvkjnkhq89N6oS8mgw01smEwBL0umvQp36&keywords=Ajazz+ak870){: .btn .btn-blue }
+
+[Тайпинг](https://www.youtube.com/shorts/k3Y6AowoDo8){: .btn .btn-purple }

### Xinmeng m87

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2015.png)

-- Материал корпуса: ABS Пластик
-- Количество клавиш: 87
-- Тип крепления: Gasket mount
-- Свитчи: Outemu White Jade
-- Хотсвап:3/5 pin
-- Плейт: Поликарбонат
-- Подключение: bt/2.4g/провод
-- Подсветка: rgb
-- Софт: Проприетарный
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>ABS Пластик</dd>
+  <dt>Количество клавиш</dt>
+  <dd>87</dd>
+  <dt>Тип крепления</dt>
+  <dd>Gasket mount</dd>
+  <dt>Свитчи</dt>
+  <dd>Outemu White Jade</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Плейт</dt>
+  <dd>Поликарбонат</dd>
+  <dt>Подключение</dt>
+  <dd>bt/2.4g/провод</dd>
+  <dt>Подсветка</dt>
+  <dd>rgb</dd>
+  <dt>Софт</dt>
+  <dd>Проприетарный</dd>
+</dl>

Честно если, не вижу смысла описывать по сути очередную тклку в этом бюджете, потому что она особо не будет отличаться от вышеперечисленных. Просто нормальная ткл-ка.

-[Ссылка (озон)](https://www.ozon.ru/product/xinmeng-klaviatura-besprovodnaya-m87prov2-angliyskaya-raskladka-sirenevyy-lilovyy-1934635930/?at=83tBgM4YViRvAYpxi467RzxSREjPkKCQNyx2Zi7rDpDE&keywords=xinmeng+m87)
-
-[Тайпинг](https://www.youtube.com/shorts/7QIbleXXRPo)
+[Ссылка (озон)](https://www.ozon.ru/product/xinmeng-klaviatura-besprovodnaya-m87prov2-angliyskaya-raskladka-sirenevyy-lilovyy-1934635930/?at=83tBgM4YViRvAYpxi467RzxSREjPkKCQNyx2Zi7rDpDE&keywords=xinmeng+m87){: .btn .btn-blue }
+
+[Тайпинг](https://www.youtube.com/shorts/7QIbleXXRPo){: .btn .btn-purple }

## **98%/Fullsize:**

@@ -282,58 +426,94 @@

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2016.png)

-- Материал корпуса: ABS Пластик
-- Количество клавиш: 98
-- Тип крепления: gasket mount
-- Свитчи: на выбор
-- Хотсвап: 3/5 pin
-- Плейт: поликарбонат
-- Подключение:  bt/2.4g/провод
-- Подсветка: Есть
-- Аккумулятор: 8000mAh
-- Софт: Проприетарный
-
-[Ссылка (озон)](https://www.ozon.ru/product/ajazz-igrovaya-klaviatura-besprovodnaya-ak980-russkaya-raskladka-belyy-1693324695/?at=z6tOzyx3qcMG7j2XuqBmjPmTynzRPOhx7oXGwc6nZyJ9&keywords=ajazz+ak980)
-
-[Тайпинг](https://www.youtube.com/shorts/BnPcuXjhpP0)
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>ABS Пластик</dd>
+  <dt>Количество клавиш</dt>
+  <dd>98</dd>
+  <dt>Тип крепления</dt>
+  <dd>gasket mount</dd>
+  <dt>Свитчи</dt>
+  <dd>на выбор</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Плейт</dt>
+  <dd>поликарбонат</dd>
+  <dt>Подключение</dt>
+  <dd>bt/2.4g/провод</dd>
+  <dt>Подсветка</dt>
+  <dd>Есть</dd>
+  <dt>Аккумулятор</dt>
+  <dd>8000mAh</dd>
+  <dt>Софт</dt>
+  <dd>Проприетарный</dd>
+</dl>
+
+[Ссылка (озон)](https://www.ozon.ru/product/ajazz-igrovaya-klaviatura-besprovodnaya-ak980-russkaya-raskladka-belyy-1693324695/?at=z6tOzyx3qcMG7j2XuqBmjPmTynzRPOhx7oXGwc6nZyJ9&keywords=ajazz+ak980){: .btn .btn-blue }
+
+[Тайпинг](https://www.youtube.com/shorts/BnPcuXjhpP0){: .btn .btn-purple }

### Aula f99

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2017.png)

-- Материал корпуса: ABS Пластик
-- Количество клавиш: 99
-- Тип крепления: gasket  mount
-- Плейт: поликарбонат
-- Переключатели: Leobog Graywood V3 \ Leobog Reaper
-- Хотсвап: 3/5 pin
-- Подключение: bt/2.4g/провод
-- Подсветка: Есть
-- Аккумулятор: 8000mAh
-- Софт: Проприетарный
-
-[Ссылка (озон)](https://www.ozon.ru/product/aula-igrovaya-klaviatura-besprovodnaya-besprovodnaya-mehanicheskaya-klaviatura-aula-f99-98-bluetooth-1794738969/?at=BrtzNm28QfMD8PNMhLJwXy3FXoERY4ugxG794CWEPrn&keywords=aula+f99)
-
-[Тайпинг](https://www.youtube.com/shorts/aoYYH2FjrU4)
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>ABS Пластик</dd>
+  <dt>Количество клавиш</dt>
+  <dd>99</dd>
+  <dt>Тип крепления</dt>
+  <dd>gasket  mount</dd>
+  <dt>Плейт</dt>
+  <dd>поликарбонат</dd>
+  <dt>Переключатели</dt>
+  <dd>Leobog Graywood V3 \ Leobog Reaper</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Подключение</dt>
+  <dd>bt/2.4g/провод</dd>
+  <dt>Подсветка</dt>
+  <dd>Есть</dd>
+  <dt>Аккумулятор</dt>
+  <dd>8000mAh</dd>
+  <dt>Софт</dt>
+  <dd>Проприетарный</dd>
+</dl>
+
+[Ссылка (озон)](https://www.ozon.ru/product/aula-igrovaya-klaviatura-besprovodnaya-besprovodnaya-mehanicheskaya-klaviatura-aula-f99-98-bluetooth-1794738969/?at=BrtzNm28QfMD8PNMhLJwXy3FXoERY4ugxG794CWEPrn&keywords=aula+f99){: .btn .btn-blue }
+
+[Тайпинг](https://www.youtube.com/shorts/aoYYH2FjrU4){: .btn .btn-purple }

### Xinmeng x98

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2018.png)

-- Материал корпуса: ABS Пластик
-- Количество клавиш: 99
-- Тип крепления: gasket  mount
-- Плейт: поликарбонат
-- Переключатели: Outemu White Jade
-- Хотсвап: 3/5 pin
-- Подключение: bt/2.4g/провод
-- Подсветка: Есть
-- Аккумулятор: 4600mAh
-- Софт: Проприетарный
-
-[Ссылка (озон)](https://www.ozon.ru/product/xinmeng-igrovaya-klaviatura-besprovodnaya-x98pro-deepsea-outemu-white-angliyskaya-raskladka-1709288331/?at=GRt29lXAxTm893VVfVO9515sNXgzQAHMLJnn6hW4np5G&keywords=xinmeng+x98)
-
-[Тайпинг](https://www.youtube.com/shorts/vK3HtYLstDg)
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>ABS Пластик</dd>
+  <dt>Количество клавиш</dt>
+  <dd>99</dd>
+  <dt>Тип крепления</dt>
+  <dd>gasket  mount</dd>
+  <dt>Плейт</dt>
+  <dd>поликарбонат</dd>
+  <dt>Переключатели</dt>
+  <dd>Outemu White Jade</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Подключение</dt>
+  <dd>bt/2.4g/провод</dd>
+  <dt>Подсветка</dt>
+  <dd>Есть</dd>
+  <dt>Аккумулятор</dt>
+  <dd>4600mAh</dd>
+  <dt>Софт</dt>
+  <dd>Проприетарный</dd>
+</dl>
+
+[Ссылка (озон)](https://www.ozon.ru/product/xinmeng-igrovaya-klaviatura-besprovodnaya-x98pro-deepsea-outemu-white-angliyskaya-raskladka-1709288331/?at=GRt29lXAxTm893VVfVO9515sNXgzQAHMLJnn6hW4np5G&keywords=xinmeng+x98){: .btn .btn-blue }
+
+[Тайпинг](https://www.youtube.com/shorts/vK3HtYLstDg){: .btn .btn-purple }

# Клавиатуры в бюджете 5-8к:

@@ -343,21 +523,32 @@

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2019.png)

-- Материал корпуса: Алюминий
-- Количество клавиш: 65
-- Тип крепления: Gasket mount
-- Свитчи: Dark alien lime shaft
-- Хотсвап: 3/5 pin
-- Плейт: FR4
-- Подключение: bt/2.4g/провод
-- Подсветка: Есть
-- Софт: Нет инфы
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>Алюминий</dd>
+  <dt>Количество клавиш</dt>
+  <dd>65</dd>
+  <dt>Тип крепления</dt>
+  <dd>Gasket mount</dd>
+  <dt>Свитчи</dt>
+  <dd>Dark alien lime shaft</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Плейт</dt>
+  <dd>FR4</dd>
+  <dt>Подключение</dt>
+  <dd>bt/2.4g/провод</dd>
+  <dt>Подсветка</dt>
+  <dd>Есть</dd>
+  <dt>Софт</dt>
+  <dd>Нет инфы</dd>
+</dl>

Довольно безысвестная шисятпятка, по своей сути похожа на шугар/лаки. Первая цена на клавиатуру с фр4 плейтом в алюминиевом кейсе. Сочетание интересное, особенно за эту стоимость.

-[Ссылка (озон)](https://www.ozon.ru/product/klaviatura-besprovodnaya-dark-alien-r65-angliyskaya-raskladka-chernyy-1775303474/?at=lRt6l3AyEIJmvZBlU0yl6jjCzWmx3pIv3nn25CQpAlN1&keywords=dark+alien+r65)
-
-[Тайпинг](https://www.youtube.com/shorts/jvSmFQFjvl8)
+[Ссылка (озон)](https://www.ozon.ru/product/klaviatura-besprovodnaya-dark-alien-r65-angliyskaya-raskladka-chernyy-1775303474/?at=lRt6l3AyEIJmvZBlU0yl6jjCzWmx3pIv3nn25CQpAlN1&keywords=dark+alien+r65){: .btn .btn-blue }
+
+[Тайпинг](https://www.youtube.com/shorts/jvSmFQFjvl8){: .btn .btn-purple }

## **75%:**

@@ -365,26 +556,40 @@

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2020.png)

-- Материал корпуса: Алюминий
-- Количество клавиш: 83
-- Тип крепления: gasket mount
-- Плейт: Полипропилен\FR4
-- Переключатели: HMX Violet \ JWK Wob
-- Хотсвап: 3/5 pin
-- Кейкапы: Cherry профиль, PBT
-- Шумоизоляция: 5 слоев (Включая PET)
-- Подключение: Tri-Mode
-- Емкость аккумулятора: 3500\7000 мАч
-- Подсветка: зависит от выбранной конфигурации
-- Софт: VIA
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>Алюминий</dd>
+  <dt>Количество клавиш</dt>
+  <dd>83</dd>
+  <dt>Тип крепления</dt>
+  <dd>gasket mount</dd>
+  <dt>Плейт</dt>
+  <dd>Полипропилен\FR4</dd>
+  <dt>Переключатели</dt>
+  <dd>HMX Violet \ JWK Wob</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Кейкапы</dt>
+  <dd>Cherry профиль, PBT</dd>
+  <dt>Шумоизоляция</dt>
+  <dd>5 слоев (Включая PET)</dd>
+  <dt>Подключение</dt>
+  <dd>Tri-Mode</dd>
+  <dt>Емкость аккумулятора</dt>
+  <dd>3500\7000 мАч</dd>
+  <dt>Подсветка</dt>
+  <dd>зависит от выбранной конфигурации</dd>
+  <dt>Софт</dt>
+  <dd>VIA</dd>
+</dl>

Уже классическая бюджетная алюминиевая клавиатура, которая заевоевала сердца большого количества пользователей. Есть три модели: Lite/standart/pro, отличия на фото ниже. От себя - просто хороший вариант 75% клавиатуры, классика жанра.

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2021.png)

-[Ссылка (озон)](https://www.ozon.ru/product/wobkey-klaviatura-provodnaya-rainy-75-angliyskaya-raskladka-serebristyy-zerkalnyy-1748638672/?at=LZtl2O8Qxsm6qzkphQYMmjxtvVBzQoUXG56o1IWXVw2P&keywords=wobkey+rainy+75)
-
-[Тайпинг](https://www.youtube.com/watch?v=fHuZ3X2nX3s&ab_channel=WOBKEY)
+[Ссылка (озон)](https://www.ozon.ru/product/wobkey-klaviatura-provodnaya-rainy-75-angliyskaya-raskladka-serebristyy-zerkalnyy-1748638672/?at=LZtl2O8Qxsm6qzkphQYMmjxtvVBzQoUXG56o1IWXVw2P&keywords=wobkey+rainy+75){: .btn .btn-blue }
+
+[Тайпинг](https://www.youtube.com/watch?v=fHuZ3X2nX3s&ab_channel=WOBKEY){: .btn .btn-purple }

### Bridge 75

@@ -392,108 +597,170 @@

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2023.png)

-- Материал корпуса: Алюминий
-- Количество клавиш: 83
-- Тип крепления: gasket mount
-- Плейт: Поликарбонат\FR4
-- Переключатели: MMD Princess
-- Хотсвап: 3/5 pin
-- Кейкапы: Cherry профиль, PBT
-- Шумоизоляция: 5 слоев (Включая PET)
-- Подключение: Tri-Mode
-- Емкость аккумулятора: 4000\8000 мАч
-- Подсветка: зависит от выбранной конфигурации
-- Софт: VIA
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>Алюминий</dd>
+  <dt>Количество клавиш</dt>
+  <dd>83</dd>
+  <dt>Тип крепления</dt>
+  <dd>gasket mount</dd>
+  <dt>Плейт</dt>
+  <dd>Поликарбонат\FR4</dd>
+  <dt>Переключатели</dt>
+  <dd>MMD Princess</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Кейкапы</dt>
+  <dd>Cherry профиль, PBT</dd>
+  <dt>Шумоизоляция</dt>
+  <dd>5 слоев (Включая PET)</dd>
+  <dt>Подключение</dt>
+  <dd>Tri-Mode</dd>
+  <dt>Емкость аккумулятора</dt>
+  <dd>4000\8000 мАч</dd>
+  <dt>Подсветка</dt>
+  <dd>зависит от выбранной конфигурации</dd>
+  <dt>Софт</dt>
+  <dd>VIA</dd>
+</dl>

Не менее популярный вариант, у которого, к тому же, система петель, что позволяет проще разбирать клавиатуру. Так называемого пинга нет, так что с этой точки зрения можно не переживать. Бытует мнение, что бридж стал вторым хитом после рейни. Также есть разные версии, в том числе магнитная.

-[Ссылка (озон)](https://www.ozon.ru/product/igrovaya-klaviatura-besprovodnaya-bridge-75-anodirovannyy-mehanicheskaya-klaviatura-angliyskaya-1884252347/?at=EqtkjGXprcQBP4rrUKVYPGRtllOn7MFlrZJQgh3g9N57&keywords=Swagkeys+Bridge+75)
-
-[Тайпинг](https://www.youtube.com/watch?v=WzJkXybIDIA&ab_channel=LumeKeebs)
+[Ссылка (озон)](https://www.ozon.ru/product/igrovaya-klaviatura-besprovodnaya-bridge-75-anodirovannyy-mehanicheskaya-klaviatura-angliyskaya-1884252347/?at=EqtkjGXprcQBP4rrUKVYPGRtllOn7MFlrZJQgh3g9N57&keywords=Swagkeys+Bridge+75){: .btn .btn-blue }
+
+[Тайпинг](https://www.youtube.com/watch?v=WzJkXybIDIA&ab_channel=LumeKeebs){: .btn .btn-purple }

### Attackshark x75

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2024.png)

-- Материал корпуса: Алюминий
-- Количество клавиш: 83
-- Тип крепления: Gasket mount
-- Свитчи: CS Matcha green/Kailh blue
-- Хотсвап: 3/5 pin
-- Плейт: Поликарбонат
-- Подключение: Bt/2.4g/провод
-- Подсветка: Есть
-- Софт: VIA
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>Алюминий</dd>
+  <dt>Количество клавиш</dt>
+  <dd>83</dd>
+  <dt>Тип крепления</dt>
+  <dd>Gasket mount</dd>
+  <dt>Свитчи</dt>
+  <dd>CS Matcha green/Kailh blue</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Плейт</dt>
+  <dd>Поликарбонат</dd>
+  <dt>Подключение</dt>
+  <dd>Bt/2.4g/провод</dd>
+  <dt>Подсветка</dt>
+  <dd>Есть</dd>
+  <dt>Софт</dt>
+  <dd>VIA</dd>
+</dl>

У этой клавиатуры выделю 10000 махов аккумулятор, быструю зарядку, в остальном - очередной клон рейни. 

-[Ссылка (озон)](https://www.ozon.ru/product/mehanicheskaya-igrovaya-klaviatura-attack-shark-x75-coffee-rgb-rus-cnc-via-besprovodnaya-81-knopok-1674250244/?at=k2tov0G5Vu99jBBEu87lLlXcL3nXjOUDLwWwZSZjAmrK&keywords=Attackshark+x75)
-
-[Тайпинг](https://youtu.be/T1X5g3joNY4?si=5b3EA_VJ5VfcCUt9&t=115)
+[Ссылка (озон)](https://www.ozon.ru/product/mehanicheskaya-igrovaya-klaviatura-attack-shark-x75-coffee-rgb-rus-cnc-via-besprovodnaya-81-knopok-1674250244/?at=k2tov0G5Vu99jBBEu87lLlXcL3nXjOUDLwWwZSZjAmrK&keywords=Attackshark+x75){: .btn .btn-blue }
+
+[Тайпинг](https://youtu.be/T1X5g3joNY4?si=5b3EA_VJ5VfcCUt9&t=115){: .btn .btn-purple }

### Leobog hi8se

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2025.png)

-- Материал корпуса: Алюминий
-- Количество клавиш: 80
-- Тип крепления: Gasket mount
-- Свитчи: На выбор
-- Хотсвап: 3/5 pin
-- Плейт: FR4/ Полипроприлен
-- Подключение: Bt/2.4g/провод
-- Подсветка: Есть
-- Софт: Проприетарный
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>Алюминий</dd>
+  <dt>Количество клавиш</dt>
+  <dd>80</dd>
+  <dt>Тип крепления</dt>
+  <dd>Gasket mount</dd>
+  <dt>Свитчи</dt>
+  <dd>На выбор</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Плейт</dt>
+  <dd>FR4/ Полипроприлен</dd>
+  <dt>Подключение</dt>
+  <dd>Bt/2.4g/провод</dd>
+  <dt>Подсветка</dt>
+  <dd>Есть</dd>
+  <dt>Софт</dt>
+  <dd>Проприетарный</dd>
+</dl>

Тоже самое, что и аттакшарк, только от компании леобог. Без веса, но тоже с быстрой зарядкой.

-[Ссылка (али)](https://aliexpress.ru/item/1005007635768790.html?sku_id=12000044197920973&spm=a2g2w.productlist.search_results.2.693b6511Eq4mOM)
-
-[Тайпинг](https://www.youtube.com/watch?v=-LYhmKxz__E&ab_channel=IlIlIl)
+[Ссылка (али)](https://aliexpress.ru/item/1005007635768790.html?sku_id=12000044197920973&spm=a2g2w.productlist.search_results.2.693b6511Eq4mOM){: .btn .btn-blue }
+
+[Тайпинг](https://www.youtube.com/watch?v=-LYhmKxz__E&ab_channel=IlIlIl){: .btn .btn-purple }

### Chillkey nd75

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2026.png)

-- Материал корпуса: Алюминий
-- Количество клавиш: 82
-- Тип крепления: top mount \ silicagel partcle \ split O-Ring
-- Плейт: Поликарбонат
-- Переключатели: Gateron EF Dopamine Blue
-- Хотсвап: 3/5 pin
-- Кейкапы: Cherry профиль, PBT
-- Подключение: bt/2.4g/провод
-- Емкость аккумулятора: 3600 мАч
-- Подсветка: Есть
-- Софт: Проприетарный
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>Алюминий</dd>
+  <dt>Количество клавиш</dt>
+  <dd>82</dd>
+  <dt>Тип крепления</dt>
+  <dd>top mount \ silicagel partcle \ split O-Ring</dd>
+  <dt>Плейт</dt>
+  <dd>Поликарбонат</dd>
+  <dt>Переключатели</dt>
+  <dd>Gateron EF Dopamine Blue</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Кейкапы</dt>
+  <dd>Cherry профиль, PBT</dd>
+  <dt>Подключение</dt>
+  <dd>bt/2.4g/провод</dd>
+  <dt>Емкость аккумулятора</dt>
+  <dd>3600 мАч</dd>
+  <dt>Подсветка</dt>
+  <dd>Есть</dd>
+  <dt>Софт</dt>
+  <dd>Проприетарный</dd>
+</dl>

А вот тут уже есть что сказать. Быстросъемы, три вида маунта, экран. Отличный пребилд для тех, что хочет поиграться с маунтами и попробовать подобрать для себя что то конкретное. Единсвенное, что выделю -  не очень удобно снимать топкейс при условии топмаунта. На мой взгляд - лучший вариант маунта - оринг.

-[Ссылка (озон)](https://www.ozon.ru/product/igrovaya-klaviatura-chilkey-nd75-bluetooth-2-4g-provodnoe-trehrezhimnoe-soedinenie-mehanicheskaya-s-1918429826/?at=w0tgnQryMTlp51LVfJ87ZpvcD9Y9w3fWlOp43u4n8Amr&keywords=chilkey+nd75)
-
-[Тайпинг](https://www.youtube.com/watch?v=_NsYmZkS8HU&ab_channel=LumeKeebs)
+[Ссылка (озон)](https://www.ozon.ru/product/igrovaya-klaviatura-chilkey-nd75-bluetooth-2-4g-provodnoe-trehrezhimnoe-soedinenie-mehanicheskaya-s-1918429826/?at=w0tgnQryMTlp51LVfJ87ZpvcD9Y9w3fWlOp43u4n8Amr&keywords=chilkey+nd75){: .btn .btn-blue }
+
+[Тайпинг](https://www.youtube.com/watch?v=_NsYmZkS8HU&ab_channel=LumeKeebs){: .btn .btn-purple }

### Vgn VXE75 v2

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2027.png)

-- Материал корпуса: Алюминий
-- Количество клавиш: 82
-- Тип крепления: gasket mount
-- Плейт: Поликарбонат\FR4
-- Переключатели: Kailh BOX Iceberry Cream PRO \ VGN Ania
-- Хотсвап: 3/5 pin
-- Кейкапы: OEM профиль, PBT
-- Подключение: bt/2.4g/провод
-- Емкость аккумулятора: 3000\6000 мАч
-- Подсветка: Есть
-- Софт: Проприетарный
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>Алюминий</dd>
+  <dt>Количество клавиш</dt>
+  <dd>82</dd>
+  <dt>Тип крепления</dt>
+  <dd>gasket mount</dd>
+  <dt>Плейт</dt>
+  <dd>Поликарбонат\FR4</dd>
+  <dt>Переключатели</dt>
+  <dd>Kailh BOX Iceberry Cream PRO \ VGN Ania</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Кейкапы</dt>
+  <dd>OEM профиль, PBT</dd>
+  <dt>Подключение</dt>
+  <dd>bt/2.4g/провод</dd>
+  <dt>Емкость аккумулятора</dt>
+  <dd>3000\6000 мАч</dd>
+  <dt>Подсветка</dt>
+  <dd>Есть</dd>
+  <dt>Софт</dt>
+  <dd>Проприетарный</dd>
+</dl>

Очередной клон дождливо75, о котором особо нечего сказать. Просто очередная хорошая 75% клавиатура.

-[Ссылка (озон)](https://www.ozon.ru/product/vxe-igrovaya-klaviatura-besprovodnaya-vgn-vxe75-v2-angliyskaya-raskladka-serebristyy-1691295572/?at=QktJLqZoGcl7w6rXHMM2n0ZsAwZ3AYirJ1OrQulznw65&keywords=Vgn+VXE75+v3)
-
-[Тайпинг](https://youtu.be/q0jwm1pIZIo?si=mqPzPhPccvVFFOqy&t=128)
+[Ссылка (озон)](https://www.ozon.ru/product/vxe-igrovaya-klaviatura-besprovodnaya-vgn-vxe75-v2-angliyskaya-raskladka-serebristyy-1691295572/?at=QktJLqZoGcl7w6rXHMM2n0ZsAwZ3AYirJ1OrQulznw65&keywords=Vgn+VXE75+v3){: .btn .btn-blue }
+
+[Тайпинг](https://youtu.be/q0jwm1pIZIo?si=mqPzPhPccvVFFOqy&t=128){: .btn .btn-purple }

## **80% (TKL):**

@@ -503,23 +770,34 @@

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2028.png)

-- Материал корпуса: Алюминий
-- Количество клавиш: 88
-- Тип крепления: Gasket mount
-- Свитчи: На выбор
-- Хотсвап: 3/5 pin
-- Плейт: Полипропилен
-- Подключение: bt/2.4g/провод
-- Подсветка: Есть
-- Софт: Проприетарный
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>Алюминий</dd>
+  <dt>Количество клавиш</dt>
+  <dd>88</dd>
+  <dt>Тип крепления</dt>
+  <dd>Gasket mount</dd>
+  <dt>Свитчи</dt>
+  <dd>На выбор</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Плейт</dt>
+  <dd>Полипропилен</dd>
+  <dt>Подключение</dt>
+  <dd>bt/2.4g/провод</dd>
+  <dt>Подсветка</dt>
+  <dd>Есть</dd>
+  <dt>Софт</dt>
+  <dd>Проприетарный</dd>
+</dl>

Дешевая алюминиевая ткл’ка, без via, к сожалению, но зато дешего :D

Гаскет маунт, интересные свитчи, дешего (сколько раз еще надо написать, что дешего?)

-[Ссылка (озон)](https://aliexpress.ru/item/1005007721122147.html?sku_id=12000041971139710&spm=a2g2w.productlist.search_results.2.783e1e81QYBgay)
-
-[Тайпинг](https://www.youtube.com/shorts/D-rdzmbcJeI)
+[Ссылка (озон)](https://aliexpress.ru/item/1005007721122147.html?sku_id=12000041971139710&spm=a2g2w.productlist.search_results.2.783e1e81QYBgay){: .btn .btn-blue }
+
+[Тайпинг](https://www.youtube.com/shorts/D-rdzmbcJeI){: .btn .btn-purple }

## **95/Fullsize:**

@@ -529,37 +807,59 @@

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2029.png)

-- Материал корпуса: ABS Пластик
-- Количество клавиш: 98
-- Тип крепления: Gasket mount
-- Свитчи: Akko V3 Piano Pro
-- Хотсвап: 3/5 pin
-- Плейт: Поликарбонат
-- Подключение: bt/2.4g/провод
-- Подсветка: Есть
-- Софт: Проприетарный
-
-[Ссылка (озон)](https://www.ozon.ru/product/akko-klaviatura-besprovodnaya-5098b-akko-v3-piano-pro-chernyy-zolotoy-1752556335/?at=vQtrZP2G8cAVXrJ8Tw0x0QYHoLv6BMh3B706ZiAPoREA&keywords=akko+5098B)
-
-[Тайпинг](https://www.youtube.com/shorts/qvJUOGC-klw)
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>ABS Пластик</dd>
+  <dt>Количество клавиш</dt>
+  <dd>98</dd>
+  <dt>Тип крепления</dt>
+  <dd>Gasket mount</dd>
+  <dt>Свитчи</dt>
+  <dd>Akko V3 Piano Pro</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Плейт</dt>
+  <dd>Поликарбонат</dd>
+  <dt>Подключение</dt>
+  <dd>bt/2.4g/провод</dd>
+  <dt>Подсветка</dt>
+  <dd>Есть</dd>
+  <dt>Софт</dt>
+  <dd>Проприетарный</dd>
+</dl>
+
+[Ссылка (озон)](https://www.ozon.ru/product/akko-klaviatura-besprovodnaya-5098b-akko-v3-piano-pro-chernyy-zolotoy-1752556335/?at=vQtrZP2G8cAVXrJ8Tw0x0QYHoLv6BMh3B706ZiAPoREA&keywords=akko+5098B){: .btn .btn-blue }
+
+[Тайпинг](https://www.youtube.com/shorts/qvJUOGC-klw){: .btn .btn-purple }

### VGN V98 Pro V2

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2030.png)

-- Материал корпуса: ABS Пластик
-- Количество клавиш: 99
-- Тип крепления: Gasket mount
-- Свитчи: На выбор, в зависимости от цвета
-- Хотсвап: 3/5 pin
-- Плейт: Поликарбонат
-- Подключение: bt/2.4g/провод
-- Подсветка: Есть
-- Софт: Проприетарный
-
-[Ссылка (озон)](https://www.ozon.ru/product/mehanicheskaya-klaviatura-vgn-v98-pro-v2-svich-steam-wave-pro-tsvet-black-currant-2-4g-bt-1773766288/?at=16tLX0k1GHnEDvRBs7zV3kMSVPzrjotgBw2PqH3jW6Dv&keywords=VGN+V98+PRO+V2)
-
-[Тайпинг](https://youtu.be/ripu_Yq_Fcw?si=LzL8clF73Lr_Qd7j&t=693)
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>ABS Пластик</dd>
+  <dt>Количество клавиш</dt>
+  <dd>99</dd>
+  <dt>Тип крепления</dt>
+  <dd>Gasket mount</dd>
+  <dt>Свитчи</dt>
+  <dd>На выбор, в зависимости от цвета</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Плейт</dt>
+  <dd>Поликарбонат</dd>
+  <dt>Подключение</dt>
+  <dd>bt/2.4g/провод</dd>
+  <dt>Подсветка</dt>
+  <dd>Есть</dd>
+  <dt>Софт</dt>
+  <dd>Проприетарный</dd>
+</dl>
+
+[Ссылка (озон)](https://www.ozon.ru/product/mehanicheskaya-klaviatura-vgn-v98-pro-v2-svich-steam-wave-pro-tsvet-black-currant-2-4g-bt-1773766288/?at=16tLX0k1GHnEDvRBs7zV3kMSVPzrjotgBw2PqH3jW6Dv&keywords=VGN+V98+PRO+V2){: .btn .btn-blue }
+
+[Тайпинг](https://youtu.be/ripu_Yq_Fcw?si=LzL8clF73Lr_Qd7j&t=693){: .btn .btn-purple }

# Клавиатуры в бюджете 8-12к:

@@ -569,21 +869,32 @@

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2031.png)

-- Материал корпуса: Алюминий
-- Количество клавиш: 67
-- Тип крепления: Gasket mount
-- Свитчи: Weikav coconut latte
-- Хотсвап: 3/5 pin
-- Плейт: FR4
-- Подключение: bt/2.4g/провод
-- Подсветка: Есть
-- Софт: VIA
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>Алюминий</dd>
+  <dt>Количество клавиш</dt>
+  <dd>67</dd>
+  <dt>Тип крепления</dt>
+  <dd>Gasket mount</dd>
+  <dt>Свитчи</dt>
+  <dd>Weikav coconut latte</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Плейт</dt>
+  <dd>FR4</dd>
+  <dt>Подключение</dt>
+  <dd>bt/2.4g/провод</dd>
+  <dt>Подсветка</dt>
+  <dd>Есть</dd>
+  <dt>Софт</dt>
+  <dd>VIA</dd>
+</dl>

Первая версия нашумела как дешевый алюм кит 65% клавиатуры, сейчас же вторая версия - доработанный кит с возможностью покупки пребилдом. Неплохого качества алюм кейса, есть выбор по кейкапам, имеется вес. Если нужна 65% клавиатура - хороший вариант.

-[Ссылка (али)](https://aliexpress.ru/item/1005006994521622.html?sku_id=12000045889497321&spm=a2g2w.productlist.search_results.0.30519877i2CVfJ)
-
-[Тайпинг](https://www.youtube.com/shorts/mShoMl6D3Tc)
+[Ссылка (али)](https://aliexpress.ru/item/1005006994521622.html?sku_id=12000045889497321&spm=a2g2w.productlist.search_results.0.30519877i2CVfJ){: .btn .btn-blue }
+
+[Тайпинг](https://www.youtube.com/shorts/mShoMl6D3Tc){: .btn .btn-purple }

## **75%:**

@@ -593,91 +904,146 @@

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2032.png)

-- Материал корпуса: Алюминий
-- Количество клавиш: 81
-- Тип крепления: Gasket mount
-- Свитчи: Womier pom linear
-- Хотсвап: 3/5 pin
-- Плейт: Полипропилен
-- Подключение: bt/2.4g/провод
-- Подсветка: Есть
-- Софт: VIA
-
-[Ссылка (али)](https://aliexpress.ru/item/1005007297748592.html?sku_id=12000040108331976&spm=a2g2w.productlist.search_results.1.324cdaf78lAOKl)
-
-[Тайпинг](https://www.youtube.com/shorts/r-8uLpm2f-0)
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>Алюминий</dd>
+  <dt>Количество клавиш</dt>
+  <dd>81</dd>
+  <dt>Тип крепления</dt>
+  <dd>Gasket mount</dd>
+  <dt>Свитчи</dt>
+  <dd>Womier pom linear</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Плейт</dt>
+  <dd>Полипропилен</dd>
+  <dt>Подключение</dt>
+  <dd>bt/2.4g/провод</dd>
+  <dt>Подсветка</dt>
+  <dd>Есть</dd>
+  <dt>Софт</dt>
+  <dd>VIA</dd>
+</dl>
+
+[Ссылка (али)](https://aliexpress.ru/item/1005007297748592.html?sku_id=12000040108331976&spm=a2g2w.productlist.search_results.1.324cdaf78lAOKl){: .btn .btn-blue }
+
+[Тайпинг](https://www.youtube.com/shorts/r-8uLpm2f-0){: .btn .btn-purple }

### Furycube f75

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2033.png)

-- Материал корпуса: Алюминий
-- Количество клавиш: 88
-- Тип крепления: Gasket mount
-- Свитчи: На выбор
-- Хотсвап: 3/5 pin
-- Плейт: Поликарбонат
-- Подключение: bt/2.4g/провод
-- Подсветка: Есть
-- Софт: Нет информации
-
-[Ссылка (али)](https://aliexpress.ru/item/1005007277258939.html?sku_id=12000040039323033&spm=a2g2w.productlist.search_results.1.54575667W31bPq)
-
-[Тайпинг](https://www.youtube.com/shorts/Dc9_3Wtd9Ew)
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>Алюминий</dd>
+  <dt>Количество клавиш</dt>
+  <dd>88</dd>
+  <dt>Тип крепления</dt>
+  <dd>Gasket mount</dd>
+  <dt>Свитчи</dt>
+  <dd>На выбор</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Плейт</dt>
+  <dd>Поликарбонат</dd>
+  <dt>Подключение</dt>
+  <dd>bt/2.4g/провод</dd>
+  <dt>Подсветка</dt>
+  <dd>Есть</dd>
+  <dt>Софт</dt>
+  <dd>Нет информации</dd>
+</dl>
+
+[Ссылка (али)](https://aliexpress.ru/item/1005007277258939.html?sku_id=12000040039323033&spm=a2g2w.productlist.search_results.1.54575667W31bPq){: .btn .btn-blue }
+
+[Тайпинг](https://www.youtube.com/shorts/Dc9_3Wtd9Ew){: .btn .btn-purple }

### Yunzii al75

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2034.png)

-- Материал корпуса: Алюминий
-- Количество клавиш: 83
-- Тип крепления: Gasket mount
-- Свитчи: Blue grey tactile/cocoa cream v2
-- Хотсвап: 3/5 pin
-- Плейт: Поликарбонат
-- Подключение: bt/2.4g/провод
-- Подсветка: Есть
-- Софт: Проприетарный
-
-[Ссылка (али)](https://aliexpress.ru/item/1005007728852536.html?sku_id=12000042832626806&spm=a2g2w.productlist.search_results.4.53807d8f2my66Y)
-
-[Тайпинг](https://www.youtube.com/shorts/mEvLP0GdOGo)
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>Алюминий</dd>
+  <dt>Количество клавиш</dt>
+  <dd>83</dd>
+  <dt>Тип крепления</dt>
+  <dd>Gasket mount</dd>
+  <dt>Свитчи</dt>
+  <dd>Blue grey tactile/cocoa cream v2</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Плейт</dt>
+  <dd>Поликарбонат</dd>
+  <dt>Подключение</dt>
+  <dd>bt/2.4g/провод</dd>
+  <dt>Подсветка</dt>
+  <dd>Есть</dd>
+  <dt>Софт</dt>
+  <dd>Проприетарный</dd>
+</dl>
+
+[Ссылка (али)](https://aliexpress.ru/item/1005007728852536.html?sku_id=12000042832626806&spm=a2g2w.productlist.search_results.4.53807d8f2my66Y){: .btn .btn-blue }
+
+[Тайпинг](https://www.youtube.com/shorts/mEvLP0GdOGo){: .btn .btn-purple }

### YUNZII AL80

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2035.png)

-- Материал корпуса: Алюминий
-- Количество клавиш: 79
-- Тип крепления: Gasket mount
-- Свитчи: На выбор
-- Хотсвап: 3/5 pin
-- Плейт: Поликарбонат
-- Подключение: bt/2.4g/провод
-- Подсветка: Есть
-- Софт: Проприетарный
-
-[Ссылка (озон)](https://www.ozon.ru/product/yunzii-igrovaya-klaviatura-besprovodnaya-al80-angliyskaya-raskladka-chernyy-1922640879/?at=pZtpZL6ArC0zBpxAUr6nBWBuxDpqVAfAgjWvli5rmAlw&keywords=Yunzii+al75)
-
-[Тайпинг](https://www.youtube.com/shorts/MERznpx1-4s)
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>Алюминий</dd>
+  <dt>Количество клавиш</dt>
+  <dd>79</dd>
+  <dt>Тип крепления</dt>
+  <dd>Gasket mount</dd>
+  <dt>Свитчи</dt>
+  <dd>На выбор</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Плейт</dt>
+  <dd>Поликарбонат</dd>
+  <dt>Подключение</dt>
+  <dd>bt/2.4g/провод</dd>
+  <dt>Подсветка</dt>
+  <dd>Есть</dd>
+  <dt>Софт</dt>
+  <dd>Проприетарный</dd>
+</dl>
+
+[Ссылка (озон)](https://www.ozon.ru/product/yunzii-igrovaya-klaviatura-besprovodnaya-al80-angliyskaya-raskladka-chernyy-1922640879/?at=pZtpZL6ArC0zBpxAUr6nBWBuxDpqVAfAgjWvli5rmAlw&keywords=Yunzii+al75){: .btn .btn-blue }
+
+[Тайпинг](https://www.youtube.com/shorts/MERznpx1-4s){: .btn .btn-purple }

### Epomaker p75

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2036.png)

-- Материал корпуса: Алюминий
-- Количество клавиш: 81
-- Тип крепления: Gasket mount
-- Свитчи: На выбор
-- Хотсвап: 3/5 pin
-- Плейт: Поликарбонат
-- Подключение: bt/2.4g/провод
-- Подсветка: Есть
-- Софт: Проприетарный
-
-[Ссылка (озон)](https://www.ozon.ru/product/epomaker-klaviatura-p75-angliyskaya-raskladka-belyy-1922833426/?at=oZt6mGWvNTVY894qHOnLXRWCqljrpyi9A9nBlSklXKxn&keywords=Epomaker+p75)
-
-[Тайпинг](https://www.youtube.com/shorts/973CCj01gOM)
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>Алюминий</dd>
+  <dt>Количество клавиш</dt>
+  <dd>81</dd>
+  <dt>Тип крепления</dt>
+  <dd>Gasket mount</dd>
+  <dt>Свитчи</dt>
+  <dd>На выбор</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Плейт</dt>
+  <dd>Поликарбонат</dd>
+  <dt>Подключение</dt>
+  <dd>bt/2.4g/провод</dd>
+  <dt>Подсветка</dt>
+  <dd>Есть</dd>
+  <dt>Софт</dt>
+  <dd>Проприетарный</dd>
+</dl>
+
+[Ссылка (озон)](https://www.ozon.ru/product/epomaker-klaviatura-p75-angliyskaya-raskladka-belyy-1922833426/?at=oZt6mGWvNTVY894qHOnLXRWCqljrpyi9A9nBlSklXKxn&keywords=Epomaker+p75){: .btn .btn-blue }
+
+[Тайпинг](https://www.youtube.com/shorts/973CCj01gOM){: .btn .btn-purple }

## **80% (TKL):**

@@ -685,21 +1051,32 @@

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2037.png)

-- Материал корпуса: Алюминий
-- Количество клавиш: 88
-- Тип крепления: Gasket mount
-- Свитчи: Kailh cocoa
-- Хотсвап: 3/5 pin
-- Плейт: FR4/Алюминий
-- Подключение: bt/2.4g/провод
-- Подсветка: Есть
-- Софт: VIA
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>Алюминий</dd>
+  <dt>Количество клавиш</dt>
+  <dd>88</dd>
+  <dt>Тип крепления</dt>
+  <dd>Gasket mount</dd>
+  <dt>Свитчи</dt>
+  <dd>Kailh cocoa</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Плейт</dt>
+  <dd>FR4/Алюминий</dd>
+  <dt>Подключение</dt>
+  <dd>bt/2.4g/провод</dd>
+  <dt>Подсветка</dt>
+  <dd>Есть</dd>
+  <dt>Софт</dt>
+  <dd>VIA</dd>
+</dl>

А вот эта клавиатура нашумела в первом квартале 25 года. Отличный ткл пребилд со всех точек зрения: отличная обработка алюминия, в разных цветах - разные варианты покрытия (Есть екоат и анодирование). Конструктивно - рейни с быстросъемами. Отличного качества кейкапы, свитчи для тех, кто хочет мощно подолбить, что в купе с вариативностью по плейтам (а в комплекте кладут алюминиевый и FR4) дает возможность выбрать тот саунд, что ххотелось бы конкретно вам. Из плюсов - классная плата с возможностью использования нескольких видов раскладки, с возможностью установки pcb стабов и 7ю пробела. Из минусов - плейты стоковые вообще не подходят для установки пцб стабов и требуют хирургического вмешательства напильником. В продаже есть плейты для 7ю пробела.

-[Ссылка (озон)](https://www.ozon.ru/product/wobkey-klaviatura-provodnaya-wobkey-crush-80-reboot-angliyskaya-raskladka-goluboy-1749265847/?at=pZtpZL6ArCJjOMqwsZ0wZyoT9OpMzJuWEAQ3KSK4ZYRv&keywords=Crush80)
-
-[Тайпинг](https://youtu.be/7clyVhxgFSo?si=Jq9AkMzlQrhbBeof&t=850)
+[Ссылка (озон)](https://www.ozon.ru/product/wobkey-klaviatura-provodnaya-wobkey-crush-80-reboot-angliyskaya-raskladka-goluboy-1749265847/?at=pZtpZL6ArCJjOMqwsZ0wZyoT9OpMzJuWEAQ3KSK4ZYRv&keywords=Crush80){: .btn .btn-blue }
+
+[Тайпинг](https://youtu.be/7clyVhxgFSo?si=Jq9AkMzlQrhbBeof&t=850){: .btn .btn-purple }

## **95/Fullsize:**

@@ -707,21 +1084,32 @@

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2038.png)

-- Материал корпуса: Пластик
-- Количество клавиш: 97
-- Тип крепления: Gasket mount
-- Свитчи: Gateron yupiter
-- Хотсвап: 3/5 pin
-- Плейт: Поликарбонат
-- Подключение: bt/2.4g/провод
-- Подсветка: Есть
-- Софт: VIA
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>Пластик</dd>
+  <dt>Количество клавиш</dt>
+  <dd>97</dd>
+  <dt>Тип крепления</dt>
+  <dd>Gasket mount</dd>
+  <dt>Свитчи</dt>
+  <dd>Gateron yupiter</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Плейт</dt>
+  <dd>Поликарбонат</dd>
+  <dt>Подключение</dt>
+  <dd>bt/2.4g/провод</dd>
+  <dt>Подсветка</dt>
+  <dd>Есть</dd>
+  <dt>Софт</dt>
+  <dd>VIA</dd>
+</dl>

PCB Стабилизаторы с завода, отличные кейкапы, хорошее качество кейса. Кейхрон - классика жанра. Просто хороший околофуллсайз для тех, кому это необходимо.

-[Ссылка (озон)](https://www.ozon.ru/product/keychron-igrovaya-klaviatura-besprovodnaya-v5-max-3-tipa-podklyucheniya-97-klavish-tsvet-carbon-1803599371/?at=oZt6m1BMYIW7rgvmSkJrDqncVo8EgvC3y0EVnUZO0P0D&keywords=%D0%BA%D0%BB%D0%B0%D0%B2%D0%B8%D0%B0%D1%82%D1%83%D1%80%D0%B0)
-
-[Тайпинг](https://www.youtube.com/watch?v=m0wZY30q0S4&ab_channel=igfxBalance)
+[Ссылка (озон)](https://www.ozon.ru/product/keychron-igrovaya-klaviatura-besprovodnaya-v5-max-3-tipa-podklyucheniya-97-klavish-tsvet-carbon-1803599371/?at=oZt6m1BMYIW7rgvmSkJrDqncVo8EgvC3y0EVnUZO0P0D&keywords=%D0%BA%D0%BB%D0%B0%D0%B2%D0%B8%D0%B0%D1%82%D1%83%D1%80%D0%B0){: .btn .btn-blue }
+
+[Тайпинг](https://www.youtube.com/watch?v=m0wZY30q0S4&ab_channel=igfxBalance){: .btn .btn-purple }

# Клавиатуры в бюджете 12-15к:

@@ -731,43 +1119,67 @@

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2039.png)

-- Материал корпуса: Алюминий
-- Количество клавиш: 88
-- Тип крепления: Gasket/leaf-spring
-- Свитчи: **Azure Linear**
-- Хотсвап: 3/5 pin
-- Плейт: FR4/ Полипропилен
-- Подключение: bt/2.4g/провод
-- Подсветка: Есть
-- Софт: VIA
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>Алюминий</dd>
+  <dt>Количество клавиш</dt>
+  <dd>88</dd>
+  <dt>Тип крепления</dt>
+  <dd>Gasket/leaf-spring</dd>
+  <dt>Свитчи</dt>
+  <dd>**Azure Linear**</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Плейт</dt>
+  <dd>FR4/ Полипропилен</dd>
+  <dt>Подключение</dt>
+  <dd>bt/2.4g/провод</dd>
+  <dt>Подсветка</dt>
+  <dd>Есть</dd>
+  <dt>Софт</dt>
+  <dd>VIA</dd>
+</dl>

Тот же краш80, только на стероидах и от более почитаемого производителя. Есть вариативность по методу крепления плейта, те же быстросъемы, та же вариативность по плейтам. Если есть возможность докинуть от краша до эво - разницу вы почувствуете.

-[Ссылка (озон)](https://www.ozon.ru/product/evoworks-igrovaya-klaviatura-besprovodnaya-qwertykeys-evo80-azure-linear-switch-fr4-1759632795/?at=jYtZ0JNRLUBjoB2vuyr9ZZyFgzgD29INWo8DvCY8XVVo&keywords=evoworks+evo+80\)
-
-[Тайпинг](https://www.youtube.com/watch?v=B7hkH0mAX3w&ab_channel=markerchun)
+[Ссылка (озон)](https://www.ozon.ru/product/evoworks-igrovaya-klaviatura-besprovodnaya-qwertykeys-evo80-azure-linear-switch-fr4-1759632795/?at=jYtZ0JNRLUBjoB2vuyr9ZZyFgzgD29INWo8DvCY8XVVo&keywords=evoworks+evo+80\){: .btn .btn-blue }
+
+[Тайпинг](https://www.youtube.com/watch?v=B7hkH0mAX3w&ab_channel=markerchun){: .btn .btn-purple }

### Cidoo v87 pro

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2040.png)

-- Материал корпуса: Алюминий
-- Количество клавиш: 87
-- Тип крепления: gasket
-- Плейт: Поликарбонат
-- Переключатели: Quark Quake Matte
-- Хотсвап: 3/5 pin
-- Кейкапы: Cherry профиль, PBT
-- Подключение: Tri-Mode
-- Емкость аккумулятора: 3000 мАч
-- Подсветка: Есть
-- Софт: VIA
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>Алюминий</dd>
+  <dt>Количество клавиш</dt>
+  <dd>87</dd>
+  <dt>Тип крепления</dt>
+  <dd>gasket</dd>
+  <dt>Плейт</dt>
+  <dd>Поликарбонат</dd>
+  <dt>Переключатели</dt>
+  <dd>Quark Quake Matte</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Кейкапы</dt>
+  <dd>Cherry профиль, PBT</dd>
+  <dt>Подключение</dt>
+  <dd>Tri-Mode</dd>
+  <dt>Емкость аккумулятора</dt>
+  <dd>3000 мАч</dd>
+  <dt>Подсветка</dt>
+  <dd>Есть</dd>
+  <dt>Софт</dt>
+  <dd>VIA</dd>
+</dl>

Толстый алюминий, Pcb стабы в стоке, обработка корпуса песком. ТКЛ для староверов, ни дать, ни взять. Прекрасные кейкапы, отличные свитчи, хорошая сборка. Все прекрасно, но уже есть вопросы целесообразности на фоне выхода новых клавиатур.

-[Ссылка (али)](https://aliexpress.ru/item/1005005625009734.html?sku_id=12000033790632374&spm=a2g2w.productlist.search_results.0.5ce443c9B6kuRb)
-
-[Тайпинг](https://www.youtube.com/shorts/OIroudlz42E)
+[Ссылка (али)](https://aliexpress.ru/item/1005005625009734.html?sku_id=12000033790632374&spm=a2g2w.productlist.search_results.0.5ce443c9B6kuRb){: .btn .btn-blue }
+
+[Тайпинг](https://www.youtube.com/shorts/OIroudlz42E){: .btn .btn-purple }

## **95/Fullsize:**

@@ -775,22 +1187,35 @@

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2041.png)

-- Материал корпуса: Алюминий
-- Количество клавиш: 99
-- Тип крепления: gasket mount
-- Плейт: Поликарбонат\FR4
-- Переключатели: Sea Salt \ Huano Gift \ Ice cream \ KTT Hyacinth
-- Хотсвап: 3/5 pin
-- Кейкапы: Cherry профиль, PBT
-- Подключение: Провод \ Tri-Mode
-- Емкость аккумулятора: 4000 мАч
-- Подсветка: в зависимости от конфигурации
-- Софт: Проприетарный
+<dl>
+  <dt>Материал корпуса</dt>
+  <dd>Алюминий</dd>
+  <dt>Количество клавиш</dt>
+  <dd>99</dd>
+  <dt>Тип крепления</dt>
+  <dd>gasket mount</dd>
+  <dt>Плейт</dt>
+  <dd>Поликарбонат\FR4</dd>
+  <dt>Переключатели</dt>
+  <dd>Sea Salt \ Huano Gift \ Ice cream \ KTT Hyacinth</dd>
+  <dt>Хотсвап</dt>
+  <dd>3/5 pin</dd>
+  <dt>Кейкапы</dt>
+  <dd>Cherry профиль, PBT</dd>
+  <dt>Подключение</dt>
+  <dd>Провод \ Tri-Mode</dd>
+  <dt>Емкость аккумулятора</dt>
+  <dd>4000 мАч</dd>
+  <dt>Подсветка</dt>
+  <dd>в зависимости от конфигурации</dd>
+  <dt>Софт</dt>
+  <dd>Проприетарный</dd>
+</dl>

Отличная клавиатура с закосом под QK100. Экран, тип плейта и подсветка идет в зависимости от  версии. Из минусов можно выделить отсутствие поддержки PCB-Mount стабилизаторов и VIA.

![image.png](%D0%A7%D1%82%D0%BE%20%D0%BA%D1%83%D0%BF%D0%B8%D1%82%D1%8C%20%D0%B2%20%D0%B1%D1%8E%D0%B4%D0%B6%D0%B5%D1%82%D0%B5%20%D0%B4%D0%BE%2015%20%D1%82%D1%8B%D1%81%D1%8F%D1%87%20(2025)/image%2042.png)

-[Ссылка (али)](https://aliexpress.ru/item/1005006634241041.html?sku_id=12000037882011805&spm=a2g2w.productlist.search_results.12.625a9e78pmXN8q)
-
-[Тайпинг](https://www.youtube.com/shorts/hJPt5JQoh30)
+[Ссылка (али)](https://aliexpress.ru/item/1005006634241041.html?sku_id=12000037882011805&spm=a2g2w.productlist.search_results.12.625a9e78pmXN8q){: .btn .btn-blue }
+
+[Тайпинг](https://www.youtube.com/shorts/hJPt5JQoh30){: .btn .btn-purple }
*** End Patch